src/components/base/outline-alert/outline-alert.stories.ts

```diff
@@ -0,0 +1,158 @@
+import { html, TemplateResult } from 'lit';
+import './outline-alert';
+import { argTypeSlotContent } from '../../base/outline-element/utils/utils';
+import { alertSizes, alertStatusTypes } from './outline-alert';
+import { ifDefined } from 'lit/directives/if-defined';
+
+export default {
+  title: 'Molecules/Alert',
+  component: 'outline-alert',
+  argTypes: {
+    headerSlot: {
+      name: 'slot="outline-alert--header"',
+      description: 'The header slot. For example a header title.',
+      table: {
+        category: 'Slots',
+      },
+      // We aren't setting a control type here so we can edit the value using the infered object.
+    },
+    // We aren't setting a control type here so we can edit the value using the infered object.
+    defaultSlot: {
+      ...argTypeSlotContent.defaultSlot,
+      table: {
+        category: 'Slots',
+      },
+    },
+    statusType: {
+      description:
+        'The status type of the alert. Such as `information` or `warning`.',
+      options: alertStatusTypes,
+      control: { type: 'select' },
+    },
+    size: {
+      description: 'The size of the alert.',
+      options: alertSizes,
+      control: { type: 'select' },
+    },
+    shouldShowIcon: {
+      description: '(not yet implemented) Should we show the icon',
+      control: { type: 'boolean' },
+    },
+    isInteractive: {
+      description: 'Is there an interaction in the alert, such as a button.',
+      control: { type: 'boolean' },
+    },
+  },
+  parameters: {
+    docs: {
+      description: {
+        component: `
```

Comment:
Could we clean up this formatting a bit? It gets a bit difficult to follow what's going on here

Reply:
Thanks for taking a look. Could you give some examples of what's difficult?

The overall formatting is forced by Prettier and this template literal I put the markdown aligned to the left because I find that easier to read. It's like a HEREDOC / NOWDOC. Did you want these indented to the `component` property?

Reply:
I think when we tried this with indenting and a template literal, the Markdown didn't work. :( I think Markdown needs the header / etc syntax at the start of the line, `#`.